example-apps/kitchen-sink-example/source/components/change-detection/change-detection.ts

```diff
@@ -31,10 +30,6 @@ import {UserInfoCheckOnce} from './user-info-checkonce';
     <user-info-push [user]="user">
     </user-info-push>
 
-    <user-info-checkonce [user]="user">
```

Comment:
Sadly, they haven't updated the stability status of the API to deprecated yet. 😭